src/components/Post/Icon-delete.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+import { IoTrash } from "react-icons/io5";
+
+export default function IconDelete() {
+  return <IconDeleteStyle />;
+}
+const IconDeleteStyle = styled(IoTrash)`
+  color: white;
+  font-size: 19px;
+  margin-left: 9px;
+`;
```

Comment:
Niceeee

Reply:
Thank you. I still need to do the responsive.